apps/frontend/src/modules/project-api-keys/slices/actions.ts

```diff
@@ -16,13 +16,12 @@ const create = createAsyncThunk<
 	AsyncThunkConfig
 >(`${sliceName}/create`, async (payload, { dispatch, extra }) => {
 	const { projectApiKeysApi, toastNotifier } = extra;
-	const { projectId } = payload;
 
 	const response = await projectApiKeysApi.create(payload);
 
 	toastNotifier.showSuccess(NotificationMessage.PROJECT_API_KEY_CREATE_SUCCESS);
 
-	void dispatch(projectActions.getById({ id: String(projectId) }));
+	void dispatch(projectActions.updateApiKey(response.apiKey));
```

Comment:
I think we don't need a separate action for this, we return response in this action, so we can access it inside create.success case in reducer. It's ok if we use action from another module inside projects, just import it the same way as we do for components.